app_dart/test/request_handlers/github_webhook_test.dart

```diff
@@ -2051,8 +2125,7 @@ void foo() {
       expect(db.values.values.whereType<Commit>().length, 1);
     });
 
-    test('Does not test pest draft pull requests.', () async {
-      //FOR REVIEW : umm is pest a typo here or maybe some form of pr
+    test('Does not comment about needing tests on draft pull requests.', () async {
```

Comment:
"test pest" was correct (means commenting that the author needs to write a test) but confusing.

Reply:
haha thanks for explaining! I have always wondered what a test pest is